### PR TITLE
Profile update - outlook part

### PR DESCRIPTION
**Pull request by RostyslavStepanchuk**

### CODE REVIEW

File: client/src/components/UpdateProfile/UpdateProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+  FormControlLabel,
+  FormLabel,
+  Grid,
+  IconButton,
+  Radio,
+  RadioGroup,
+  TextField,
+  Typography
+} from '@material-ui/core'
+import { KeyboardDatePicker, MuiPickersUtilsProvider, } from '@material-ui/pickers'
```
Comment (tarasbashuk) [inline client/src/components/UpdateProfile/UpdateProfile.jsx]: extra coma

Comment (tarasbashuk) [reply client/src/components/UpdateProfile/UpdateProfile.jsx]: Actually I have a lot of code styling issues. Maybe we changed code style config and I haven't updated it?

File: client/src/utils/helpers/inputValidators.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+export const isNoErrors = errorsObj => {
+  return Object.values(errorsObj).some(err => err !== '')
+}
```
Comment (tarasbashuk) [inline client/src/utils/helpers/inputValidators.js:3]: I think there is a wrong logic. 
We could write either: 
`areNoErrors = errorsObj => {
  return Object.values(errorsObj).every(err => err === '')
}`
 or 
`isError = errorsObj => {
  return Object.values(errorsObj).some(err => err !== '')
}` and then
`const inputIsValid = !validate()`

File: client/src/components/UpdateProfile/UpdateProfile.jsx (hunk 1 above)
Comment (RostyslavStepanchuk) [reply client/src/components/UpdateProfile/UpdateProfile.jsx]: I did not, and neither had errors with `npm run eslint-build`
am I using wrong command?

File: client/src/utils/helpers/inputValidators.js (hunk 2 above)
Comment (RostyslavStepanchuk) [reply client/src/utils/helpers/inputValidators.js:3]: Yep, you're right, thanks

File: client/src/components/UpdateProfile/UpdateProfile.jsx (hunk 1 above)
Comment (tarasbashuk) [reply client/src/components/UpdateProfile/UpdateProfile.jsx]: I'm using `npx standart` and it shows plenty of problems

**Review by tarasbashuk — APPROVED** [review]

**Pull request merged**